client/my-sites/site-settings/manage-connection/site-ownership.jsx

```diff
@@ -59,13 +59,24 @@ class SiteOwnership extends Component {
 
 	onSelectConnectionOwner = user => {
 		const { translate } = this.props;
+		const message = (
+			<Fragment>
+				{ translate( 'Are you sure you want to transfer site ownership to {{user /}}?', {
+					components: {
+						user: <strong>{ user.display_name || user.name }</strong>,
+					},
+				} ) }
+				<br />
+				<br />
```

Comment:
Can we wrap the `{ translate() }`s in `<p />`s instead?

(We might've discussed this before, but I can't quite remember the outcome.)

Reply:
Yes, good point 👍 Done in f9b65637c6758f838bad628a68d2fe1c52e9859e